addons/scope_remover/config.cpp

```diff
@@ -0,0 +1,18 @@
+class CfgPatches {
+    class cnto_scope_remover {
+        units[] = {};
+        weapons[] = {};
+        requiredAddons[] = {
+            "cba_settings"
+        };
+    };
+};
+
+class CfgFunctions {
+    class cnto_scope_remover {
+        class all {
+            file = "\cnto\additions\scope_remover";
+            class init;
+        };
+    };
+};
```

Comment:
I've also just noticed that you never actually run `fn_init`, thus this addon probably doesn't work at all.
```cpp
class Extended_PreInit_EventHandlers {
    class cnto_scope_remover {
        init = "[] call cnto_scope_remover_fnc_init";
    };
};
```